.travis.yml

```diff
@@ -114,38 +114,14 @@ matrix:
 
         # Try on Windows
         - os: windows
-          env: PYTHON_VERSION=3.7 SETUP_CMD='test --readonly'
```

Comment:
Removing `--readonly` seems unrelated. Is there a need for this?

Reply:
yes, `--readonly` was a feature in v3.1, thus we cannot use here. I could have push it directly to the branch or patch the backports as I usually do, but given there was this PR, I would just added to it.

Reply:
Ah, gotcha. Thanks!